File: src/com/brianc/style/Display.java
package com.brianc.style;

public enum Display {
	INLINE, BLOCK, NONE;
	
	public static Display find(String name) {
		switch(name) {
		case "block":
			return BLOCK;
		case "none":
			return NONE;
		default:
			return INLINE;
		}
	}
}

File: src/com/brianc/style/Display.kt
package com.brianc.style

enum class Display {
    INLINE, BLOCK, NONE;


    companion object {

        fun find(name: String): Display {
            when (name) {
                "block" -> return BLOCK
                "none" -> return NONE
                else -> return INLINE
            }
        }
    }
}
